playwright/src/test/java/com/microsoft/playwright/TestPageEvaluate.java

```diff
@@ -532,16 +532,15 @@ void shouldWorkWithCSP() {
 
   @Test
   void shouldEvaluateException() {
-    try {
+    PlaywrightException e = assertThrows(PlaywrightException.class, () -> {
```

Comment:
this test is failing, looks like it used to pass due to the test not verifying that an exception is thrown

Reply:
Yeah, looks like a typo, see my suggestion below how to fix it.

Reply:
only webkit is failing now, an exception is not being thrown here

Reply:
[Upstream test](https://github.com/microsoft/playwright/blob/27ffdcc9440d9d76ddca1c8059b46ee98c42ac25/tests/page/page-evaluate.spec.ts#L575-L583) does not actually throw, it returns an error and checks that it is properly formatted. Let me align java test with the upstream one.